src/one_dragon/base/operation/application/application_group_config.py

```
@@ -0,0 +1,157 @@
+            enabled: 是否启用
+        """
+        self.app_id: str = app_id
+        self.enabled: bool = enabled
+        self.app_name: str = ''  # 不需要保存 每次注入
+
+
+class ApplicationGroupConfig(YamlOperator):
+
+    def __init__(self, instance_idx: int, group_id: str):
+        """
+        应用组配置，保存在 config/{instance_idx}/{group_id}/_group.yml 文件中
```

Ruff (0.13.2)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)